digicampipe/scripts/spe.py

```diff
@@ -206,6 +206,32 @@ def find_pulse_2(events, threshold_sigma, widths, **kwargs):
         yield event
 
 
+def find_pulse_3(events):
+    w = np.array([1, 2, 3, 4, 5, 4, 3, 2, 1], dtype=np.float32)
```

Comment:
Thanks ! this looks pretty good to speed up the pulse finding. if you put `dtype=np.int16` (or the same type as adc_samples) does this speed up even more ?

Reply:
I have to try